pkg/ddevapp/wordpress.go

```diff
@@ -78,46 +81,80 @@ const (
  ddev manages this file and may delete or overwrite the file unless this comment is removed.
  */
 
+// Include local settings if it exists.
+$docroot = getenv('NGINX_DOCROOT');
+if (file_exists($docroot . '/wp-config-ddev.php')) {
+	require_once $docroot . '/wp-config-ddev.php';
+}
+
 // ** MySQL settings - You can get this info from your web host ** //
 /** The name of the database for WordPress */
-define('DB_NAME', '{{ $config.DatabaseName }}');
+if (!defined('DB_NAME'))
+	define( 'DB_NAME', '{{ $config.DatabaseName }}' );
 
 /** MySQL database username */
-define('DB_USER', '{{ $config.DatabaseUsername }}');
+if (!defined('DB_USER'))
+	define( 'DB_USER', '{{ $config.DatabaseUsername }}' );
 
 /** MySQL database password */
-define('DB_PASSWORD', '{{ $config.DatabasePassword }}');
+if (!defined('DB_PASSWORD'))
+	define( 'DB_PASSWORD', '{{ $config.DatabasePassword }}' );
 
 /** MySQL hostname */
-define('DB_HOST', '{{ $config.DatabaseHost }}');
+if (!defined('DB_HOST'))
+ 	define( 'DB_HOST', '{{ $config.DatabaseHost }}' );
 
 /** Database Charset to use in creating database tables. */
-define('DB_CHARSET', 'utf8mb4');
+define( 'DB_CHARSET', 'utf8mb4' );
 
 /** The Database Collate type. Don't change this if in doubt. */
-define('DB_COLLATE', '');
+define( 'DB_COLLATE', '' );
 
 /**
  * WordPress Database Table prefix.
  */
-$table_prefix  = '{{ $config.TablePrefix }}';
+if(!isset($table_prefix))
```

Comment:
Not sure I understand the question 100%.

If `wp-config-ddev.php` is includes it would be coming from there. So it's falling back to `$config.TablePrefix` if not set.

You have to set the DB prefix for WP which by default is `wp_`. However a decent amount of suer have a different prefix, hosts will have unique prefixes etc.